apps/examples/wc-next10/components/PieButton.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useRef } from 'react';
+import Head from 'next/head';
+import { createComponent } from '@lit-labs/react';
+import { PieButton, BUTTON_SIZE, BUTTON_VARIANT } from '@justeattakeaway/pie-button';
+
+const PieBtn = createComponent({
+    tagName: 'pie-button',
+    elementClass: PieButton,
+    react: React,
+    events: { onCustomEvent: 'CustomEvent' },
+});
+
+export default function PButton () {
```

Comment:
We could do with breaking down this file. This is supposed to be a react component wrapping around our web component (as I understand it). However, we appear to be returning the entire application shell (the main index.html content).